_sass/_content.scss

```diff
@@ -1,8 +1,48 @@
 .content {
   margin: 0 auto;
   max-width: $medium-screen;
+
+  .content + & {
+    margin-top: 2rem;
+  }
 }
 
 .content--centered {
   text-align: center;
 }
+
+.content--grid {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 1rem;
+}
+
+.content--grid__item {
+  flex-basis: 100%;
+  flex-grow: 1;
+  flex-shrink: 0;
```

Comment:
I have felt less and less of a desire to write shorthand. I find myself becoming more verbose.